packages/editor/src/components/ComponentsList/ComponentList.tsx

```diff
@@ -40,6 +53,53 @@ function getCategoryOrder(name: string): number {
     : 0;
 }
 
+const FilterIcon = createIcon({
```

Comment:
Why not use a chakra icon?

Reply:
The chakra-icon does not appear to have a filter icon